### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by Nejjari001**
Screenshot 2023-03-11 151340
Screenshot 2023-03-11 151410

## In this Project I implemented following features:
- Login page and registration page: built with Devise.
- Navigation menu that allows users to open all of the pages you created
- Followed N+1 queries concept.

- [ ] Food list:

 - Should display a list of food added by the logged-in user as in the wireframe (for teams with only 2 people - display also quantity of a given food).
 - Should lead to a form that allows users to add new food.

- [ ] Recipes list:

 - Should display a list of recipes created by the logged-in user as in the wireframe.
 - Should lead to recipe details.
 - If the user is the owner of the recipe, should allow the user to delete it

- [ ] Public recipe list:

 - Should display a list of all public recipes ordered by newest as in the wireframe.
 - Should lead to recipe details.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,23 +187,21 @@ I would like to thank Microverse for their continued support. They've provided u
...
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./LICENSE.md) licensed.
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (BenMukebo) [inline README.md:207]: - [ ] Great job on the Recipies list, Public recipe list and Recipe details implementation 👏🏻 However their Design/Views should match the structure of the given: ([Recipies list wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_index.png?raw=true), [Public recipe list wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/public_recipe_index.png?raw=true), and [Recipe details wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true)). Kindly address this 💯 👍🏻 

image

- Note make sure you display the Public button on the Recipe details page 🖼️  👍🏻 
image

- Your Page:
image

Comment (BenMukebo) [inline README.md:207]: - [ ] Kindly make sure the app solves the N+1 problem when fetching all Recipes for a user as requested in the project requirements. For more documentation, visits the [Solving the N+1 problem in Rails](https://medium.com/idopterlabs/solving-the-n-1-problem-in-rails-b5ea677e78e0)


- [ ] Kindly, Create a navigation menu that allows users to open all of the pages you created. 👇🏻
- [ ] Let's make sure to write Unit and Integration tests ✍🏻 

image

**Review by BenMukebo — CHANGES_REQUESTED** [review]
## Hi @Nejjari001, 

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


### To Highlight 🎉
- Git flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Nejjari001, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- No linter errors ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**